resources/js/processes-catalogue/components/ProcessesCatalogue.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="d-flex w-100">
+    <div class="w-25">
+      <h4> {{ $t('Processes Browser') }} </h4>
+    </div>
+    <div class="w-100">
+      <div v-if="!fields.length" class="d-flex justify-content-center py-5 w-100">
+        <div>
+          <div class="d-flex justify-content-center my-5">
+            <img
+              class="image d-flex"
+              src="/img/processes-catalogue-empty.svg"
+              alt="recent projects"
+            >
+          </div>
+          <h4 class="text-center"> {{ $t('Currently you dont have processes created') }} </h4>
+          <p class="text-center"> {{ $t('We compel you to create new processes using our templates') }} </p>
```

Comment:
Please update the message to:
`We encourage you to create new processes using our templates`
I requested to update the mockups too